### PR TITLE
Change module references to integrations in Logs and Metrics intros

### PR DESCRIPTION
**Pull request by colleenmcginnis**
Fixes https://github.com/elastic/observability-docs/issues/1323

Changes module references to integrations in [Log monitoring](https://www.elastic.co/guide/en/observability/current/monitor-logs.html) and [Metrics monitoring](https://www.elastic.co/guide/en/observability/current/analyze-metrics.html).

### CODE REVIEW

File: docs/en/observability/analyze-metrics.asciidoc [hunk 1]
```diff
@@ -12,9 +12,9 @@ to help diagnose problematic spikes, identify high resource utilization,
 automatically discover and track pods, and unify your metrics
 with logs and APM data in {es}.
 
-Using {metricbeat} modules, you can ingest and analyze
-metrics from servers, Docker containers, Kubernetes orchestrations, explore and
-analyze Prometheus-style metrics or application telemetries, and many more.
+Using {agent} integrations, you can ingest and analyze metrics from servers, Docker containers,
+Kubernetes orchestrations, explore and analyze Prometheus-style metrics or application
```
Comment (dedemorton) [inline docs/en/observability/analyze-metrics.asciidoc:16]: According to the latest integrations documentation, Prometheus is still a technical preview, so I don't think I would mention it here.

**Review by dedemorton — COMMENTED** [review]
Minor comment about prometheus; otherwise looks good!

File: docs/en/observability/analyze-metrics.asciidoc (hunk 1 above)
Comment (colleenmcginnis) [reply docs/en/observability/analyze-metrics.asciidoc:16]:
```suggestion
Kubernetes orchestrations, explore and analyze application
```

**Review by dedemorton — APPROVED** [review]
LGTM!

**Pull request merged**